modules/nf-core/checkv/endtoend/main.nf

```diff
@@ -0,0 +1,41 @@
+process CHECKV_ENDTOEND {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::checkv=1.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/checkv:1.0.1--pyhdfd78af_0':
+        'quay.io/biocontainers/checkv:1.0.1--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path db
+
+    output:
+    tuple val(meta), path ("${prefix}")      , emit: results
+    tuple val(meta), path ("${prefix}/*.tsv"), emit: tsv
```

Comment:
Given there are lots of differen tTSVs for different hings, it might be nice to pick each one up explicitly as output channels

Reply:
Done, also included channels for the identified viruses & proviruses